cluster/gce/util.sh

```diff
@@ -1078,23 +1119,50 @@ function kube-down {
     fi
   done
 
-  # First delete the master (if it exists).
-  if gcloud compute instances describe "${MASTER_NAME}" --zone "${ZONE}" --project "${PROJECT}" &>/dev/null; then
+  local -r REPLICA_NAME=$(get-replica-name)
+
+  # Un-register the master replica from etcd and events etcd.
+  gcloud compute ssh ${REPLICA_NAME} \
```

Comment:
Similarly as above - can you please make it a function (like remove-etcd-member or sth like that)?